dev_frankfurt.env

```diff
@@ -0,0 +1,58 @@
+PRINT_ENV=dev
+RANCHER_LABEL=dev
+IMAGE_TAG=staging
+NGINX_PORT=8009
+TOMCAT_PORT=8011
+API_URL=//mf-chsdi3.dev.bgdi.ch
```

Comment:
Most url are wrong and should be ‘sys-‘

Reply:
Yes, good point. I have stopped at a point yesterday and we can correct everything in this file as we go

Reply:
At least `VECTOR_BUCKET` should be already correct. The rest is more or less copy paste from `dev.env`